doc/source/whatsnew/v0.24.0.rst

```diff
@@ -1210,6 +1210,7 @@ update the ``ExtensionDtype._metadata`` tuple to match the signature of your
 - :meth:`Series.unstack` and :meth:`DataFrame.unstack` no longer convert extension arrays to object-dtype ndarrays. Each column in the output ``DataFrame`` will now have the same dtype as the input (:issue:`23077`).
 - Bug when grouping :meth:`Dataframe.groupby()` and aggregating on ``ExtensionArray`` it was not returning the actual ``ExtensionArray`` dtype (:issue:`23227`).
 - Bug in :func:`pandas.merge` when merging on an extension array-backed column (:issue:`23020`).
+- Bug in :func:`pandas.merge` when setting None in suffixes (:issue: `24782`).
```

Comment:
be more explicit on what is changing here

Reply:
thanks, I will change later once collecting more feedbacks from other committers. @jreback